### ISSUE TITLE
Dashboard's contributors section doesn't show all the namespaces I own

### ISSUE THREAD

**Issue by Bobgy** [kubeflow/kubeflow#4569]
/kind bug

**What steps did you take and what happened:**
1. I set up a Kubeflow 0.7 cluster and get a `gongyuan` namespace.
2. I created a profile `demo` which also marks me as the owner.
3. However, in dashboard contributors section, it only shows me as owner for demo namespace, not gongyuan namespace.
![UI bug preview](https://user-images.githubusercontent.com/4957653/70771517-57a17180-1dac-11ea-86ed-35112419abec.png)

**What did you expect to happen:**
I should be able to manage all the namespaces' contributors I own in UI.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Kubeflow version: build version 0.7.0
- kfctl version: (use `kfctl version`): I no longer have the kfctl I used to deploy this cluster, but I think it's not related to the issue.
- Kubernetes platform: GKE

**Comment by Bobgy** [comment]
Actually, I'm not sure if Kubeflow intended to support one user being owner of multiple namespaces. Is that the case?

/cc @kunmingg

**Comment by Bobgy** [comment]
In addition to UI support, documentation about how to grant contributor through SDK/CLI would also mitigate the issue.

**Comment by kunmingg** [comment]
By default user can only be owner of one namespace:
* Normal users don't have permission to create profile, only admin can assign multiple namespaces.
* Kubeflow UI only show `register` page when the user doesn't own any namespace. So user only register once and get one namespace.

**Comment by avdaredevil** [comment]
Thanks for making this issue!

**Comment by Bobgy** [comment]
Estimated efforts is 1~1.5 weeks, we need the relevant updates to the 2 other components, namespace selector (trivial if any), registration flow (also perhaps trivial), and tests

Some notes for how to develop:
* First time only:
```
npm i
make build-local
npm run build
```
* Run the commands in two separate terminals: `npm run dev` and `kubectl port-forward svc/profiles-kfam 8081:8081`

To override user identity when developing locally: So I use an extension called [MOD-Header](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj?src=bewisse) that's very useful, and you can set your headers like:
```
X-Goog-Authenticated-User-Email: accounts.google.com:kubeflow-user@k
```
This will kick in multi-user isolation mode on the dashboard. You can verify your isolation mode in the payload from /api/workgroup/exists (hasAuth field).

**Comment by Bobgy** [comment]
Update, I don't think I will have cycles to fix this either. Note that central dashboard UI is written in Polymer with web components. Someone with related expertise should be able to fix it a lot faster.

Some notes to how to develop: https://github.com/kubeflow/kubeflow/issues/4569#issuecomment-599857524
The specific page we need to fix is: https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/public/components/manage-users-view.pug

**Comment by abdulsalama** [comment]
@yanniszark from the discussion on the other bug that we just closed as duplicate, you suggested this:
"@abdulsalama you can work around this by not using the Contributors UI to manage users."

What would be the alternative? I'm not aware of any other mechanism where I can add contributors to my profile other than the Contributors UI?

**Comment by abdulsalama** [comment]
ah do you mean creating the rolebinding manually?

**Comment by Bobgy** [comment]
We have documentation for adding users programmatically now: https://master.kubeflow.org/docs/components/multi-tenancy/getting-started/#managing-contributors-manually

I think that's enough for this use case.

**Comment by Bobgy** [comment]
Hmm, but this feature will still be a nice to have.

**Comment by HassanOuda** [comment]
@Bobgy Can we revive this? It would be awesome if have the manage contributors page using the namespace selector dropdown for the owned namespaces to toggle the page so that we can add contributors to each owned namespace.

**Comment by blairdrummond** [comment]
@wg102 We already have the code for this done, do we have a PR for that? 

CC @HassanOuda , I believe the code is here

https://github.com/StatCan/kubeflow/commit/01a17e6b4ac2520327d95eea6a414a234f11db5e#diff-7b12be43febc6e18259dd77c004315b4143d570234e0f15acf5f9d7b79c371b5

**Comment by wg102** [comment]
@blairdrummond The PR has not been created yet. It should be straight forward to create

**Comment by HassanOuda** [comment]
@blairdrummond and I think I needed this as well:
https://github.com/StatCan/kubeflow/commit/1fa21268306f8be26d21efe905d0f13e558169ed

Thanks so much! good work on this!

**Comment by aaron-arellano** [comment]
can this be reopened?

**Comment by thesuperzapper** [comment]
@aaron-arellano sure
/reopen

**Comment by thesuperzapper** [comment]
@wg102 (or anyone else watching), if you want to make a PR based on https://github.com/StatCan/kubeflow/commit/1fa21268306f8be26d21efe905d0f13e558169ed, I am happy to review and get it merged!

**Comment by thesuperzapper** [comment]
@kimwnasptd this seems like an easy win to get into `1.6.0.RC1`, as the code is pretty much already done in https://github.com/StatCan/kubeflow/commit/1fa21268306f8be26d21efe905d0f13e558169ed.

**Comment by TranThanh96** [comment]
v1.6.1 still not have this feature ?

**Comment by emilyyujieli** [comment]
hello @thesuperzapper ,

Kindly ask do you know is there any update for this feature? Since it seems v1.6.1 still not have it.

**Comment by yingding** [comment]
It would be nice to have this feature. Currently also with KF 1.6.1, should i be the owner of multiple namespace, it is not possible to manage contributors from all the namespaces i own directly from UI. It is still possible to create contributors manually, but it would be really a great feature to have.

**Comment by jbottum** [comment]
@kubeflow/wg-notebooks-leads Hey Notebook leads, could someone please reply to this question.

**Comment by Jason-CKY** [comment]
Is there a PR for this feature yet?

**Comment by thesuperzapper** [comment]
This feature has been added in https://github.com/kubeflow/kubeflow/pull/7582

It will be included in Kubeflow 1.9.0.
